src/integration/pages/articles/mediaPlayerCaption.js

```diff
@@ -0,0 +1,10 @@
+export default () => {
+  // eslint-disable-next-line no-only-tests/no-only-tests
+  describe.only('Media Player Caption', () => {
```

Comment:
Let's remove the lint override, and also the .only

```suggestion
  describe('Media Player Caption', () => {
```